word_guess.rb

```diff
@@ -0,0 +1,110 @@
+require "colorize"
+
+# class Image generates beating heart
+class Image
+  attr_writer :num_attempts
+  def initialize
+    @num_attempts = 5
+    @heart_parts = ["  ,d88b.d88b,  ", "  88888888888  ", "  `Y8888888Y'  ", "    `Y888Y'    ", "      `Y'      "]
+
+  end
+
+  def print_heart
+    loser_img = ""
+    @num_attempts.times do |attempt|
+      loser_img += "#{@heart_parts[attempt]}".blue.on_red.blink
+      loser_img += "\n"
+    end
+    (5 - @num_attempts).times do
+      loser_img += "               ".blue.on_red.blink
+      loser_img += "\n"
+    end
+    return loser_img
+
+  end
+end
+
+# class Game logic runs game
+class Game
+
+  attr_accessor :word, :word_split
+
+  def initialize
+    @words = ["apple", "coconut", "carrot", "skein", "metronome"]
+    @attempts = 5
+    @image = Image.new
+    @word = @words.sample
+    @word_split = @word.split('')
+    @word_in_progress = Array.new(@word.length, "_ ")
+  end
+
+  def round
+    # get word length
+    # loop through word length and puts "_" that many times
+    while @attempts != 0 && @word_split != @word_in_progress
```

Comment:
This method feels a little long to me. Could you break it up into several smaller methods? For example, you might make one that gets a letter from the user and validates it, and another to update the game state based on the user's guess.